tool.go

```diff
@@ -143,7 +143,11 @@ func main() {
 					}
 					if len(pkgs) == 1 { // Only consider writing output if single package specified.
 						if pkgObj == "" {
-							pkgObj = filepath.Base(pkg.ImportPath) + ".js"
+							if !build.IsLocalImport(pkg.ImportPath) {
+								pkgObj = filepath.Base(pkg.ImportPath) + ".js"
```

Comment:
Import paths are always separated by a forward slash, never backslash, so we should use `path.Base` rather than `filepath.Base` on it. Mind making that change?